### ISSUE TITLE
ES6 Imports remain in build when requiring recharts via ES6

### ISSUE THREAD

**Issue by lewiscowper** [recharts/react-smooth#9]
First off, thanks so much for this package, it's made so much of my working life easier with react + charts. :)

However, as of the most recent release of react-smooth, I seem to have hit an issue related to the module bundling and transforming changes.

```
ERROR in /main-60bfb053936fa59d6901.js from UglifyJs
SyntaxError: Unexpected token: name (Animate) [./~/recharts/~/react-smooth/lib/index.js:1,0]
```

We import specific charts that we need as ES6 imports, as in `import { Surface, Symbols } from 'recharts';`

We're running this through UglifyJS, and hitting a syntax error with "Animate" as shown above, in browser that translates to an `unexpected token 'import'` error. This is the index.js file as required by recharts@^0.20.2.

```
function(module, exports) {
	import Animate from './Animate';
	import { configBezier, configSpring } from './easing';
	import { translateStyle } from './util';
	import AnimateGroup from './AnimateGroup';
	
	export { configSpring, configBezier, AnimateGroup, translateStyle };
	
	export default Animate;
}
```

I noticed the recent es6 import changes to the release, and our build broke ~ 2 hours ago, around the time the new version was tagged, so I'm guessing these two things are related.

Quick fix would be pinning the version of react-smooth that recharts uses to 0.1.17, and updating that once a build process fix is found, (I've had a look around, but I'm not sure what winds up getting published to npm/required by the main 'recharts' package. If all else fails I'll fork the main package and pin the dependency and keep an eye out for an upstream fix).

**Comment by emmenko** [comment]
We're also getting this error. Works with `0.1.17`. Latest **patch** release breaks it all ;)

**Comment by xile611** [comment]
Sorry! It's my fault. I forgot to check the compiled files in `/lib`. I think it has been fixed in the version 0.1.19. Please check.

**Comment by bramus** [comment]
Verified as fixed with 0.1.19
